src/CrawlAbstractLvl/private/abslvlworld.cpp

```diff
@@ -0,0 +1,60 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the GPLv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "baff.h"
+#include "obstacleblue.h"
+#include "obstaclered.h"
+#include "abslvlcontrol.h"
+#include "abslvlworld.h"
+#include <snake.h>
+#include "Crawl/iworlditem.h"
+
+
+AbsLvlWorld::AbsLvlWorld() {
+
+}
+
+IPlayer *AbsLvlWorld::initPlayer() const {
+    return new Snake();
```

Comment:
you should create a own Snake class item. For more information see the test lvl.